sc4s_sync.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# echo all the commands
+set -x
+
+REPOORG=splunk
+if [[  $GITHUB_USER && ${GITHUB_USER-x} ]]
+then
+    echo "GITHUB_USER Found"
+else
+    echo "GITHUB_USER Not found"
+    exit 1
+fi
+if [[  $GITHUB_TOKEN && ${GITHUB_TOKEN-x} ]]
+then
+    echo "GITHUB_TOKEN Found"
+else
+    echo "GITHUB_TOKEN Not found"
+    exit 1
+fi
+
+current_tag=$(cat SC4S_matrix.conf | grep -oP '(?<=VERSION=).*')
+echo $current_tag
+
+latest_tag=$(curl -s https://api.github.com/repos/splunk/splunk-connect-for-syslog/releases/latest | grep "tag_name" | cut -d : -f 2)
+new_value=$(echo $latest_tag | rev | cut -c3- | rev)
+new_value=$(echo $new_value | cut -c3-)
+echo $new_value
+sudo apt-get install -y python3 python3.7 python3-pip
+pip install pip --upgrade
+python -m pip install packaging
+
+git checkout test/version-change
+var=`python -c "from packaging import version; print('True' if(version.parse(str('$new_value')) > version.parse(str('$current_tag'))) else 'False')"`
+echo $var
+
+if [ "$var" = "True" ];
+then
+
+    # Logic for to raise pull request
+    git config --global user.email "addonfactory@splunk.com"
+    git config --global user.name "Addon Factory template"
+    BRANCH=test/sc4s-version-update
+    git checkout -b $BRANCH   
+    sed -i "s/$current_tag/$new_value/g" SC4S_matrix.conf
+    git diff
+    git add SC4S_matrix.conf
+    git status
+    git commit -m "test: updated new $new_value version to matrix file"
+    git push -f --set-upstream origin $BRANCH
+    git log | head
+    git checkout master
+    git merge test/sc4s-version-update
```

Comment:
after merging, we also have to push the branch, right?

Reply:
Added logic to push branch to master and delete local branch.